aries-site/src/data/structures/templates/templates.js

```diff
@@ -195,6 +195,9 @@ export const templates = [
       'Shipping',
       'Customize',
       'Required and Optional Fields',
+      'Full-page form guidance',
```

Comment:
Once the headings are all finalized, let's update these to match.

Reply:
they are matched now but will change if changes come up